webClient/src/app/core/menu-bar/menu-bar.component.ts

```diff
@@ -589,9 +589,7 @@ export class MenuBarComponent implements OnInit, OnDestroy {
   saveFile() {
     let fileContext = this.editorControl.fetchActiveFile();
     if (!fileContext) {
-      this.snackBar.open('Unable to save, no file found.', 'Dismiss', {duration: MessageDuration.Medium, panelClass: 'center'});
-    } else if (fileContext.model.isDataset) {
-      this.snackBar.open('Dataset saving not yet supported.', 'Dismiss', {duration: MessageDuration.Short, panelClass: 'center'});
+      this.snackBar.open('Warning: Cannot save, no buffer found', 'Dismiss', {duration: MessageDuration.Medium, panelClass: 'center'});
```

Comment:
If this will be a message to the user, looks like it is probably better to use 'content' or 'file' instead of buffer